pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "IRKsome"
+version = "0.0.1"
+authors = [
+  {name = "Rob Kirby", email = "Robert_Kirby@baylor.edu"},
+  {name = "Jorge Marchena Menendez"},
```

Comment:
We updated an author list when we fixed the docs a couple of months ago.  Shall we use that here?

Reply:
Sure I can do that. I just used what was in the setup.py. Alternatively you could omit authors here and set a maintainers field instead

Reply:
What does that look like?  just replacing `authors` with `maintainers` (and appropriate details inside)?

Reply:
Yes. It isn't really important though. I will use the authors list defined in `conf.py`.